app/org/sagebionetworks/bridge/play/controllers/ParticipantController.java

```diff
@@ -124,15 +124,13 @@ private Result getParticipantsInternal(Study study, String offsetByString, Strin
         return okResult(page);
     }
     
-    public Result createParticipant(String verifyEmailString) throws Exception {
+    public Result createParticipant() throws Exception {
```

Comment:
This will prevent admins from creating users in the UI... I don't currently switch to the admin API to create users because I think of that as a test UI, not something for production. I would set verifyEmail to true here unless the user has the admin role. This will allow me to create users through our tool.